agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -345,7 +347,6 @@ public static class PreviewConfiguration {
 
     public ProfilerConfiguration profiler = new ProfilerConfiguration();
     public GcEventConfiguration gcEvents = new GcEventConfiguration();
-    public AadAuthentication authentication = new AadAuthentication();
```

Comment:
can we keep here and copy over in `ConfigurationBuilder` and log warning? e.g. something like

https://github.com/microsoft/ApplicationInsights-Java/blob/383ccc9a4cb2ee03e693113c677d152cd784d2ac/agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java#L221-L232

Reply:
we have AadAuthentication.validate() that takes care of it